proposals/0029-cooperative-vector.md

```diff
@@ -318,7 +318,9 @@ interpretation** and **matrix layout** behaving as described [above]
 The base address of **matrix resource** and **matrix offset** must be 128 byte
 aligned. Also note that the size of the underlying allocation is guaranteed to
 be a multiple of 16 bytes ensuring that the 16 bytes access of the last
-row/column of the matrix is valid memory
+row/column of the matrix is valid memory. Implementations may write to the
```

Comment:
One idea is to rewrite this as : Implemetations may need to use the padding space (e.g while using smaller types like FP16 that certain hardware might not natively support and are therefore emulated with bigger native types), so as a general rule using the padding space is undefined behavior.

Just to give the reader more information of why we have this restriction.

Reply:
I think this is more about supporting cases where there is native support for the type but it has additional alignment requirements.  I think I might leave this as it is for now, can add clarification later, but that really belongs in the user documentation.